src/bindings/python/flux/accounting/test/test_job_archive_interface.py

```diff
@@ -47,63 +57,72 @@ def setUpClass(self):
             );"""
         )
 
-        # add sample jobs to job-archive database
-        id = 100
-        userid = 1234
-        username = "user" + str(userid)
-        t_submit = 1000
-        t_sched = 1005
-        t_run = 1010
-        t_cleanup = 1015
-        t_inactive = 1020
-        for i in range(4):
-            try:
-                jobs_conn.execute(
-                    """
-                    INSERT INTO jobs (
-                        id,
-                        userid,
-                        username,
-                        ranks,
-                        t_submit,
-                        t_sched,
-                        t_run,
-                        t_cleanup,
-                        t_inactive,
-                        eventlog,
-                        jobspec,
-                        R
+        def populate_job_archive_db(jobs_conn, userid, username, ranks, num_entries):
+            nonlocal id
+            nonlocal t_submit
+            nonlocal t_sched
+            nonlocal t_run
+            nonlocal t_cleanup
+            nonlocal t_inactive
+
+            for i in range(num_entries):
+                try:
+                    jobs_conn.execute(
+                        """
+                        INSERT INTO jobs (
+                            id,
+                            userid,
+                            username,
+                            ranks,
+                            t_submit,
+                            t_sched,
+                            t_run,
+                            t_cleanup,
+                            t_inactive,
+                            eventlog,
+                            jobspec,
+                            R
+                        )
+                        VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)
+                        """,
+                        (
+                            id,
+                            userid,
+                            username,
+                            ranks,
+                            time.time() - 2000,
+                            time.time() - 1000,
+                            time.time(),
+                            time.time() + 1000,
+                            time.time() + 2000,
```

Comment:
Oops. `t_submit` and the other `t_*` variables were accidentally left over from when I converted to using `time.time()`. I will append a `fixup` commit that removes those variables since they're not used.